lib/node/lib/index.d.ts

```diff
@@ -0,0 +1,47 @@
+declare module "didkit" {
+  declare interface Ed25519Key {
+    kty: string;
+    crv: string;
+    x: string;
+    d: string;
+  }
+
+  declare type Key = Ed25519Key | any;
+
+  declare interface Options {
+    proofPurpose?: string;
+    verificationMethod?: string;
+    challenge?: string;
+    domain?: string;
```

Comment:
There also may be a property `created` - should it be included too?
```
created?: string;
```

Reply:
Yeah, fixed.